starterkit_app/lib/core/presentation/views/view_life_cycle_handler.dart

```diff
@@ -14,7 +14,7 @@ import 'package:starterkit_app/core/reflection.dart';
 import 'package:starterkit_app/core/service_locator.dart';
 
 abstract final class ViewLifeCycleHandler {
-  static NavigationObserver get _navigationObserver => ServiceLocator.instance<NavigationObserver>();
+  static final NavigationObserver _navigationObserver = ServiceLocator.instance<NavigationObserver>();
```

Comment:
In the PR diff, there's a Codecov mention that the changed line is not covered by tests.
Perfect :-)
That's one of the configurations you changed in the starterkit, right?

Reply:
Yes. I recently added Codecov.